### ISSUE TITLE
Handle 'no-unused-expressions' violations due to use of Chai expect

### ISSUE THREAD

**Issue by ianwalter** [cypress-io/eslint-plugin-cypress#3]
Using a Chai expect expression in a test with the StandardJS ESLint config will throw the ESLint error:

```
error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
```

Since Chai is bundled with Cypress I think it only makes sense that eslint-plugin-cypress handle this without the user having to install a separate plugin to handle Chai expressions.

**Comment by jennifer-shehane** [comment]
Can you give a short example of the test with expression you wrote that errored so we can test this?

**Comment by ianwalter** [comment]
Sure:
```js
const onRequest = ({ request }) => {
  expect(request.body.email.billingUpdate).to.be.true
  expect(request.body.phone.documentUpdate).to.be.false
}
```

**Comment by Strajk** [comment]
👋Here's discussion about this topic https://github.com/eslint/eslint/issues/2102

**Comment by jennifer-shehane** [comment]
Eslint decided to not address this issue as it is library specific. It seems that the best fix for this would be to add the [eslint-plugin-chai-friendly](https://www.npmjs.com/package/eslint-plugin-chai-friendly) plugin to our eslint config or some similar configuration to ignore chai's `no-unused-expressions`. @chrisbreiding thoughts?

**Comment by corysimmons** [comment]
@jennifer-shehane Just tested this out. Works great and still catches errors not related to Cypress/Chai. 👍

**Comment by chrisbreiding** [comment]
Because of the way ESLint plugins work, we can't bundle a plugin with our plugin and have it "just work". The user still needs to install the chai plugin manually as their own dependency. If we include the rule and a user doesn't install the plugin (which they might not even need if they're not using the `no-unused-expressions` rule), they'll get an error saying there's an unknown rule in their configuration. 

So instead, I'm going to document this in the README.

**Comment by bmulholland** [comment]
Resurrecting an old issue, but there is now a way to bundle plugins together: https://www.npmjs.com/package/@rushstack/eslint-patch

As used by airbnb, for example: https://github.com/vuejs/eslint-config-airbnb/blob/main/packages/eslint-config-airbnb/CHANGELOG.md#700
